docs/source/tracking.rst

```diff
@@ -429,6 +429,44 @@ Autologging captures the following information:
++------------------------------------------+------------------------------------------------------------+-------------------------------------------------------------------------------------+---------------+-----------------------------------------------------------------------------------------------------------------------------------------------+
+|``pytorch-lightning trainer``             | Training loss; validation loss;average_test_accuracy;      | ``fit()`` parameters; optimizer name; learning rate; epsilon.                       | --            | Model summary on training start; `MLflow Model <https://mlflow.org/docs/latest/models.html>`_ (Pytorch model) on training end;                |
+|                                          | user-defined-metrics.                                      |                                                                                     |               | Also facilitates logging of torchscript models;                                                                                               |
+|                                          |                                                            |                                                                                     |               | Optional Documents such as ExtraFiles,requirements.txt on Training End could also be logged;                                                  |
+|                                          |                                                            |                                                                                     |               |                                                                                                                                               |
+|                                          |                                                            |                                                                                     |               |                                                                                                                                               |
++------------------------------------------+------------------------------------------------------------+-------------------------------------------------------------------------------------+---------------+-----------------------------------------------------------------------------------------------------------------------------------------------+
+|``trainer.callbacks.earlystopping``       |Training loss;validation loss;average_test_accuracy;        | ``fit()`` parameters; optimizer name; learning rate; epsilon                        | --            | Model summary on training start; `MLflow Model <https://mlflow.org/docs/latest/models.html>`_ (Pytorch model) on training end;                |
```

Comment:
Should this be pytorch_lightning.callbacks.early_stopping? https://pytorch-lightning.readthedocs.io/en/stable/early_stopping.html#early-stopping-based-on-metric-using-the-earlystopping-callback

I don't think a `trainer.callbacks.earlystopping` module exists

Reply:
Yes. The change has been incorporated.